cms/static/js/certificates/collections/signatories.js

```diff
@@ -0,0 +1,16 @@
+define([
+    'backbone', 'js/certificates/models/signatory'
+],
+function(Backbone, Signatory) {
+    'use strict';
+    console.log("signatory_collection.start");
+    var SignatoryCollection = Backbone.Collection.extend({
+        model: Signatory,
+        url: '/certificates/edX/DemoX/Demo_Course/signatories'
```

Comment:
It seems to me that we cannot have a static course URL here -- is it possible to make it dynamic?